egs/wsj/s5/steps/nnet3/lstm/make_configs.py

```diff
@@ -28,6 +28,8 @@ def ParseSpliceString(splice_indexes, label_delay=None):
     if label_delay is not None:
         left_context = -label_delay
         right_context = label_delay
+    if preserve_state == "true":
```

Comment:
left_context of the network is the amount of context necessary for computing one output value. I don't see how using a state-preservation during training is effecting this value.